.github/workflows-src/steps.lib.yml

```diff
@@ -103,6 +102,7 @@ with:
 name: "Validate"
 shell: bash
 run: |
+  echo "${{ toJson(github.event) }}"
```

Comment:
This step was randomly failing, and I can't get it to fail anymore, so leaving it in here for now.